modules/codegen/src/smithy4s/codegen/Renderable.scala

```diff
@@ -43,8 +43,7 @@ object Renderable {
 
   implicit def tupleRenderable[A](implicit
       A: Renderable[A]
-  ): Renderable[(String, A)] = (t: (String, A)) =>
-    A.render(t._2).addImport(t._1)
+  ): Renderable[(String, A)] = (t: (String, A)) => A.render(t._2)
```

Comment:
I'm a bit suspicious of this. The LHS of the tuple isn't used anymore, which means that this week's implicit is probably useless and its current usesites should be amended

Reply:
That's fair, I will go through and cleanup the usesites

Reply:
> this week's implicit

:joy:

Reply:
Lol, that's what happens when I look at github on the phone whilst being sleep deprived 😄